[2024-11-02T13:29:42.078+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-02-06T00:00:00+00:00 [queued]>
[2024-11-02T13:29:42.091+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-02-06T00:00:00+00:00 [queued]>
[2024-11-02T13:29:42.092+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T13:29:42.108+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-02-06 00:00:00+00:00
[2024-11-02T13:29:42.113+0000] {standard_task_runner.py:57} INFO - Started process 3169 to run task
[2024-11-02T13:29:42.118+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-02-06T00:00:00+00:00', '--job-id', '1224', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmp2po3d2ti']
[2024-11-02T13:29:42.122+0000] {standard_task_runner.py:85} INFO - Job 1224: Subtask submit_schemas
[2024-11-02T13:29:42.173+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-02-06T00:00:00+00:00 [running]> on host 5e812a6a7c02
[2024-11-02T13:29:42.263+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhtranquang' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-02-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-06T00:00:00+00:00'
[2024-11-02T13:29:42.264+0000] {pinot_schema_operator.py:33} ERROR - An error occured: 'function' object has no attribute 'glob'
[2024-11-02T13:29:42.274+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240206T000000, start_date=20241102T132942, end_date=20241102T132942
[2024-11-02T13:29:42.330+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T13:29:42.350+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
